cli/internal/helm/charts/edgeless/constellation-services/charts/csi-gcp-pd/values.yaml

```diff
@@ -0,0 +1,26 @@
+image:
+  csiProvisioner:
+    repo: k8s.gcr.io/sig-storage/csi-provisioner
+    tag: "v3.1.0"
+    pullPolicy: IfNotPresent
+  csiAttacher:
+    repo: k8s.gcr.io/sig-storage/csi-attacher
+    tag: "v3.4.0"
+    pullPolicy: IfNotPresent
+  csiResizer:
+    repo: k8s.gcr.io/sig-storage/csi-resizer
+    tag: "v1.4.0"
+    pullPolicy: IfNotPresent
+  csiSnapshotter:
+    repo: k8s.gcr.io/sig-storage/csi-snapshotter
+    tag: "v4.0.1"
+    pullPolicy: IfNotPresent
+  csiNodeRegistrar:
+    repo: k8s.gcr.io/sig-storage/csi-node-driver-registrar
+    tag: "v2.5.0"
+    pullPolicy: IfNotPresent
+  gcepdDriver:
+    repo: ghcr.io/edgelesssys/constellation/gcp-csi-driver
+    # CSI driver version is independent of Constellation releases
+    tag: "v1.0.0"
+    pullPolicy: IfNotPresent
```

Comment:
I would prefer to keep this information in the `versions.go` file and prefill these values in the CLI loader.

Enforcing the presence of the value could be done with a values.schema.json.